src/CSF.cpp

```diff
@@ -115,7 +115,7 @@ void CSF::do_filtering(std::vector<int>& groundIndexes,
                        std::vector<int>& offGroundIndexes,
                        bool              exportCloth) {
     // Terrain
-    std::cout << "[" << this->index << "] Configuring terrain..." << std::endl;
+    // std::cout << "[" << this->index << "] Configuring terrain..." << std::endl;
```

Comment:
It depends on perception needs.
- I can add a configuration parameter to enable it if needed.
- Or just delete this.

Reply:
@bexultandeepx 
Thanks!

> It is not going to be in production anyway if its commented out. So to enable it you need to recompile.

Btw, by "I can add a configuration parameter to enable it if needed." I meant to:
- Add a ROS parameter to the perception SWC to enable CSF debug logs. 
- And:
  - Either [modify the `CSF` class](https://github.com/DeepX-inc/CSF/blob/master/src/CSF.h#L64) to receive this on construction.
  - Or [modify the `do_filtering` function](https://github.com/DeepX-inc/prj_oliver/blob/61aeac66e991b05b7228628dd07dca5b40e00c5d/src/swc/perception/include/oliver_perception/utils/ground_filtering.hpp#L167) to accept this parameter.


If it is relevant for perception, I can add the parameter.
Otherwise, the easiest is to just delete this.

Any option is fine with me.

Reply:
From the looks of it these are some trivial logs ,however  some of  these debug parameter might be handy for quick debug. I would recommend to just comment it out for now (not completely deleting it ).Creating a flag to control it is also fine with us if it not troublesome

Reply:
There are some remaining issues likely to be related to CSF such as these: [issue1](https://app.clickup.com/t/863gwu4t9) , [issue2](https://app.clickup.com/t/320tk1e), [issue3](https://app.clickup.com/t/37hzqk1) ..etc. Although these debug messages are trivial , it could be helpful for the new engineers if they work on it . We probably meed [better debugging strategy](https://app.clickup.com/t/864ec4v64) across perception pipeline later when we have time

Reply:
Okay, I will try that with a parameter. Probably during next sprint.

Reply:
@matias-pavez-dx Thanks for your help !